php-checks/src/main/java/org/sonar/php/checks/FormattingStandardCheck.java

```diff
@@ -79,18 +82,21 @@
   @RuleProperty(
     key = "open_curly_brace_classes_functions",
     defaultValue = "true",
+    description = "Enables the validation of the open curly brace positions for function and class declarations.",
     type = "BOOLEAN")
   public boolean isOpenCurlyBraceForClassAndFunction = true;
 
   @RuleProperty(
     key = "open_curly_brace_control_structures",
     defaultValue = "true",
+    description = "Enables the validation of the open curly brace positions for control structures.",
     type = "BOOLEAN")
   public boolean isOpenCurlyBraceForControlStructures = true;
 
   @RuleProperty(
     key = "closing_curly_brace",
     defaultValue = "true",
+    description = "Enables the validation of the closing curly brace positions.",
```

Comment:
"Enables the verification that closing curly braces which follow keywords are on the same line as the keyword."